instrumentation/jdbc-mariadb-1.3.0/build.gradle

```diff
@@ -12,7 +12,7 @@ jar {
 }
 
 verifyInstrumentation {
-    passesOnly 'org.mariadb.jdbc:mariadb-java-client:[1.3.0,)'
+    passesOnly 'org.mariadb.jdbc:mariadb-java-client:[1.3.0,2.7.2]'
```

Comment:
You should use the `)` closing parenthesis to cap the range so that it's obvious which version the instrumentation fails on. 

So if it starts failing at 3.0.0 the following indicates that it supports up to, but not including, 3.0.0.

```
passesOnly 'org.mariadb.jdbc:mariadb-java-client:[1.3.0,3.0.0)'
```